src/build.py

```diff
@@ -783,7 +767,14 @@ def LLVM():
 
   proc.check_call(command, cwd=LLVM_OUT_DIR, env=cc_env)
   proc.check_call(['ninja', '-v'] + jobs, cwd=LLVM_OUT_DIR, env=cc_env)
-  proc.check_call(['ninja', 'check-all'], cwd=LLVM_OUT_DIR, env=cc_env)
+
+  def RunWithUnixUtils(cmd, **kwargs):
+    if IsWindows():
+      return Git(['bash'] + cmd, **kwargs)
+    else:
+      return proc.check_call(cmd, **kwargs)
+
+  RunWithUnixUtils(['ninja', 'check-all'], cwd=LLVM_OUT_DIR, env=cc_env)
```

Comment:
This looks slightly odd as a local function that only gets used once.
Though it seems like something that can be used elsewhere in the future.
So it probably makes more sense for it to just be a top-level function.